.github/workflows/build.yml

```diff
@@ -0,0 +1,109 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  release:
+    types:
+      - created
+  pull_request:
+
+jobs:
+  build_win:
+    runs-on: windows-latest
+
+    steps:
+    - uses: microsoft/setup-msbuild@v1.0.0
+
+    - uses: nuget/setup-nuget@v1
+
+    - uses: actions/checkout@v2
+
+    - name: build
+      working-directory: ./windows
+      run: |
+        nuget restore
+        msbuild "QMK Toolbox.sln" /verbosity:minimal /p:Configuration=Release
+        iscc install_compiler.iss
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: "windows/QMK Toolbox/bin/Release/qmk_toolbox.exe"
+    - uses: actions/upload-artifact@v2
+      with:
+        path: "windows/qmk_toolbox_install.exe"
+  
+  build_macos:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: install packagesbuild
+      working-directory: ./osx
+      run: |
+        wget http://s.sudre.free.fr/Software/files/Packages.dmg
```

Comment:
Can we do Homebrew stuff with Actions? There's a Cask for this, IIRC.

Reply:
Yeah, couldnt find it when i went looking. Got a link and I can look into it?